### ISSUE TITLE
NSF Award Numbers in 'funding' slot of EML are not specified as being NSF

### ISSUE THREAD

**Issue by jagoldstein** [NCEAS/metacatui#922]
This UX enhancement was recommended by @mpsaloha .

A great feature would be to make the funding numbers on dataset landing pages clickable, so that it opens a page with award overview information, such as [this one](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1546024&HistoricalAwards=false).

But, at a minimum, a message along the lines of "NSF, if not otherwise noted", next to the display of the `funding` slot in the metadata view, should clear up any confusion when there are only 7 digits listed without an explicit funder name.

**Comment by laurenwalker** [comment]
We should probably be inserting that information into the metadata itself instead of just the display. Although I agree a link to the NSF award website is a good feature to add to the UI.

What format would work well for the ADC? e.g. `NSF {award-number}`?

**Comment by jagoldstein** [comment]
Yes, that format makes sense to me. (Of course, a small minority of arcticdata.io holdings are NOT funded by NSF, so we should continue to accommodate those as well.)

**Comment by laurenwalker** [comment]
We could use that format when someone uses the NSF award lookup/autocomplete list. Otherwise, the editor could just add the award info to the EML exactly as the user entered it.

**Comment by jagoldstein** [comment]
Ok, but the award lookup isn't currently working, correct? So users are just entering 7 digits without any reference to "NSF".

@mpsaloha you brought this issue up, what do you think?

**Comment by csjx** [comment]
However we format it, we should be looking ahead and be able to programmatically transform our EML 2.1.1 documents to the better structure in EML 2.2.0: https://github.com/NCEAS/eml/blob/9972508f20446ada90b79dc2103c5589a2897254/xsd/eml-project.xsd#L447

**Comment by mbjones** [comment]
Agreed with @csjx let's deal with this when we refactor to support EML 2.2.0 award structures.

**Comment by jagoldstein** [comment]
@mpsaloha note that this issue you identified is being punted ^
